src/ApplicationLogs/LogReader.cs

```diff
@@ -54,6 +59,19 @@ protected override void OnSystemLoaded(NeoSystem system)
         {
             if (system.Settings.Network != Settings.Default.Network) return;
             RpcServerPlugin.RegisterMethods(this, Settings.Default.Network);
+
+            // It is possible to have dos attack by sending a lot of transactions.
+            void Ev(object _, LogEventArgs e)
+            {
+                if (e.ScriptContainer is not Transaction tx) return;
+                if (!_logEvents.TryGetValue(tx.Hash, out var list))
+                {
+                    list = new();
+                    _logEvents.Add(tx.Hash, list);
+                }
+                list.Add(e);
+            }
+            ApplicationEngine.Log += Ev;
```

Comment:
Should we add a setting in config.json to control whether logs are saved or not? I think maybe not all nodes want to do this.

Reply:
The term `log` and `notify` is abusively miss used, I don't think the user could tell which is log and which is notify.